sources/packages/api/src/services/application/application.service.ts

```diff
@@ -519,6 +506,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .leftJoin("location.institution", "institution")
       .leftJoin("institution.institutionType", "institutionType")
       .innerJoin("application.student", "student")
```

Comment:
since only student and users details are needed now, you can remove the remaining joins